scripts/train/train.py

```diff
@@ -226,10 +183,14 @@ def main(cfg: DictConfig):
     fsdp_config: Optional[Dict] = om.to_container(
         fsdp_dict_config
     ) if fsdp_dict_config is not None else None  # type: ignore
-    lora_config: Optional[DictConfig] = pop_config(cfg,
+    lora_config: Optional[DictConfig] = pop_config(model_config,
                                                    'lora',
                                                    must_exist=False,
                                                    default_value=None)
+    if lora_config is not None:
+        if lora_config.get('rank', None) is not None:
```

Comment:
What is supposed to happen if lora config is provided but rank is none? Should that be an error?